src/main/java/teammates/sqllogic/core/FeedbackResponseCommentsLogic.java

```diff
@@ -0,0 +1,42 @@
+package teammates.sqllogic.core;
+
+import java.util.UUID;
+
+import teammates.storage.sqlapi.FeedbackResponseCommentsDb;
+import teammates.storage.sqlentity.FeedbackResponseComment;
+
+/**
+ * Handles operations related to feedback response comments.
+ *
+ * @see FeedbackResponseComment
+ * @see FeedbackResponseCommentsDb
+ */
+public final class FeedbackResponseCommentsLogic {
+
+    private static final FeedbackResponseCommentsLogic instance = new FeedbackResponseCommentsLogic();
+    private FeedbackResponseCommentsDb frcDb;
+
+    private FeedbackResponseCommentsLogic() {
+        // prevent initialization
+    }
+
+    public static FeedbackResponseCommentsLogic inst() {
+        return instance;
+    }
+
+    /**
+     * Initialize dependencies for {@code FeedbackResponseCommentsLogic}.
+     */
+    void initLogicDependencies(FeedbackResponseCommentsDb frcDb) {
```

Comment:
Please also update the method `initializeDependencies` in `LogicStarter` before we forget

Reply:
updated!